### PR TITLE
initial mpp support with js and jvm

### LINKED ISSUE #8: Support Js target

### PR DESCRIPTION
**Pull request by i-walker**
resolves #8

### CODE REVIEW

File: src/commonMain/kotlin/core/Either.kt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package io.kotest.assertions.arrow.core
+package core
```
Comment (sksamuel) [inline src/commonMain/kotlin/core/Either.kt:1]: Should the FQN just be core here ?

Comment (i-walker) [reply src/commonMain/kotlin/core/Either.kt:1]: ups, gonna change that. I missed it in the refactor

File: .editorconfig [hunk 2]
```diff
@@ -9,4 +9,4 @@ trim_trailing_whitespace = true
 insert_final_newline = true
 
 [*.gradle.kts]
-indent_size = 3
+indent_size = 2
```
Comment (sksamuel) [inline .editorconfig:12]: You didn't like my 3 :)

File: src/commonMain/kotlin/io/kotest/extensions/core/Validated.kt [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package io.kotest.assertions.arrow.core
+package io.kotest.extensions.core
```
Comment (sksamuel) [inline src/commonMain/kotlin/io/kotest/extensions/core/Validated.kt:1]: Do we want to change all the package names ?
These are assertions not extensions ?

File: .editorconfig (hunk 2 above)
Comment (i-walker) [reply .editorconfig:12]: spotted haha

File: src/commonMain/kotlin/io/kotest/extensions/core/Validated.kt (hunk 3 above)
Comment (i-walker) [reply src/commonMain/kotlin/io/kotest/extensions/core/Validated.kt]: fixed :)

**Review by sksamuel — APPROVED** [review]

**Pull request merged**